src/nupic/algorithms/KNNClassifier.py

```diff
@@ -985,11 +1013,23 @@ def _getDistances(self, inputPattern, partitionId=None):
 
 
   def finishLearning(self):
+    """
+    TODO: Somebody help me write this.
+
+    :return:
+    """
     if self.numSVDDims is not None and self._vt is None:
       self.computeSVD()
 
 
   def computeSVD(self, numSVDSamples=None, finalize=True):
+    """
+    TODO: Somebody help me write this.
+
+    :param numSVDSamples:
+    :param finalize:
+    :return:
+    """
```

Comment:
@numenta/nupic-reviewers I need help writing this. You can simply respond in a comment with a function description and I'll do the rest.